src/app/features/rpc-transaction-request/post-conditions/post-conditions-details.layout.tsx

```diff
@@ -4,16 +4,20 @@ import { styled } from 'leather-styles/jsx';
 import { ensurePostConditionWireFormat } from '@leather.io/stacks';
 import { Approver } from '@leather.io/ui';
 
-import { NoPostConditions } from '@app/components/rpc-transaction-request/post-conditions/no-post-conditions';
-import { PostConditionList } from '@app/components/rpc-transaction-request/post-conditions/post-condition-list';
-import { PostConditionModeWarning } from '@app/components/rpc-transaction-request/post-conditions/post-condition-mode-warning';
 import {
-  type RpcCallContractRequestContext,
+  type RpcStxCallContractRequestContext,
+  type RpcStxDeployContractRequestContext,
   useRpcTransactionRequestContext,
 } from '@app/features/rpc-transaction-request/rpc-transaction-request.context';
 
+import { NoPostConditions } from './no-post-conditions';
+import { PostConditionList } from './post-condition-list';
+import { PostConditionModeWarning } from './post-condition-mode-warning';
+
 export function PostConditionsDetailsLayout() {
-  const { rpcRequest } = useRpcTransactionRequestContext<RpcCallContractRequestContext>();
+  const { rpcRequest } = useRpcTransactionRequestContext<
+    RpcStxCallContractRequestContext | RpcStxDeployContractRequestContext
+  >();
```

Comment:
Is there a way we can make this generic and not know about the RPC flows that use it? If we add another 5x stacks flows, does this list of types grow?